src/evergreen/performance_results.py

```diff
@@ -0,0 +1,176 @@
+# -*- encoding: utf-8 -*-
+"""Performance results representation of evergreen."""
+from __future__ import absolute_import
+
+import copy
+
+from evergreen.base import _BaseEvergreenObject, evg_attrib, evg_short_datetime_attrib, \
+    evg_datetime_attrib
+from evergreen.util import parse_evergreen_datetime
+
+
+class TestResult(_BaseEvergreenObject):
+    """Representation of a test result from Evergreen."""
+    thread_level = evg_attrib('thread_level')
+    operations_per_second = evg_attrib('ops_per_sec_values')
+    mean_operations_per_second = evg_attrib('ops_per_sec')
+
+    def __init__(self, json, api):
+        """Create an instance of a test result."""
+        super(TestResult, self).__init__(json, api)
+
+
+class TestRun(_BaseEvergreenObject):
+    """Representation of a test run from Evergreen."""
+    workload = evg_attrib('workload')
+    test_name = evg_attrib('name')
+
+    def __init__(self, test_result, api, parent):
+        """Create an instance of a test run."""
+        super(TestRun, self).__init__(test_result, api)
+        if parent:
```

Comment:
Is there a case where parent would be `None`? Should be be setting all the attributes set below to `None` in that case? It seems like it could be frustrating to work with an object where attributes may or may not exist.

Reply:
Also, is there a reason we would want these values on the child objects? I don't seem them being used anywhere.